src/views/Submit.js

```diff
@@ -13,7 +13,16 @@ export default function Submit() {
       {/* Render multiple submission form */}
```

Comment:
Update the link at top of forms to the addStation form to be bigger and easier to see